sql/system.go

```diff
@@ -33,14 +33,14 @@ const (
 	// with special rules.
 	MaxReservedDescID ID = keys.MaxReservedDescID
 	// RootNamespaceID is the ID of the root namespace.
-	RootNamespaceID ID = 0
+	RootNamespaceID ID = keys.RootNamespaceID
 
 	// System IDs should remain <= MaxReservedDescID.
-	systemDatabaseID  ID = 1
-	namespaceTableID  ID = 2
-	descriptorTableID ID = 3
-	usersTableID      ID = 4
-	zonesTableID      ID = 5
+	systemDatabaseID  ID = keys.SystemDatabaseID
```

Comment:
Well, these unexported values are only used in `system.go`. Would be easy enough to replace those uses. `RootNamespaceID` is used in more places.